lib/src/postgrest_filter_builder.dart

```diff
@@ -19,7 +19,17 @@ class PostgrestFilterBuilder extends PostgrestTransformBuilder {
   /// ```
   PostgrestFilterBuilder not(String column, String operator, dynamic value) {
     if (value is List) {
-      appendSearchParams(column, 'not.$operator.(${_cleanFilterArray(value)})');
+      if (value is List<int> || value is List<double> || value is List<num>) {
```

Comment:
Would checking if `value` is `List<num>` be sufficient enough for this case?

Reply:
a list of ints fails the test for the list of nums.

casting in dart are complicated

Reply:
Interesting. I just tried it by myself as well, and these both were true. 

```dart
print([1, 2, 3] is List<num>); // true
print([1.1, 2.1, 3.1] is List<num>); // true
```

Maybe mine worked due to different version or something? Anyway, it sounds good to leave them!

Reply:
try

```dart
print(<int>[1, 2, 3] is List<num>);
```

Reply:
I actually got true for both. I wonder why we are getting results. But just in case, let's just keep all the checks!
```dart
print(<int>[1, 2, 3] is List<num>);
print(<double>[1, 2, 3] is List<num>);
```

Reply:
sure!